examples/interpret_mul_args.rs

```diff
@@ -0,0 +1,30 @@
+extern crate parity_wasm;
+
+use std::env::args;
+
+use parity_wasm::{interpreter, ModuleInstanceInterface, RuntimeValue};
+
+fn main() {
+    let args: Vec<_> = args().collect();
+    if args.len() < 3 {
+        println!("Usage: {} <wasm file> <exported func> [<arg>...]", args[0]);
+        return;
+    }
+    let func_name = &args[2];
+    let (_, program_args) = args.split_at(3);
+    let program_args: Vec<_>  = program_args.iter().enumerate()
+        .map(|(i, arg)| RuntimeValue::I32(arg.parse().expect(&format!("Invalid i32 arg at index {}", i))))
```

Comment:
why expect i32 only here?
you can read signature first and then try to parse command line arguments according to that signature